### DISCUSSION TITLE
Can we manually enter height width of a shape in an input box and shape will formed accordingly.

### DISCUSSION THREAD

**Discussion by robertwilli1** [pikasojs/pikaso#51] [Ideas]
I am looking to make an input box in which user after selecting the shape can enter height width of the shape manually then click enter , and shape gets formed accordingly. For Example if a user wants a rectangle  and want its height to be (100px) and width to be exactly (150px), so he can enter its height width in an input bar and the shape gets formed according to the dimensions

**Comment by raminious** [comment]
I think you are looking for `update` method https://pikaso.app/api/classes/ShapeModel.html#update
